server/utils/helper.js

```diff
@@ -0,0 +1,102 @@
+// helper methods for db access
```

Comment:
Change the name of this file to be more descriptive.  Perhaps `db-helper.js`

Reply:
Fixed it! :)